lstchain/_version_cache.py

```diff
@@ -1 +1 @@
-version='0.4.5.post23+gite9b21b4'
+version='0.4.5.post189+git1dc69fb'
```

Comment:
Is this file supposed to be under version control?

Reply:
no, it shoud be removed.

Reply:
it is there:
https://github.com/cta-observatory/cta-lstchain/blob/f4c8df9d59c170b8863fa9dd74864fef6a43fd83/.gitignore#L2

actually you added it :D

Reply:
removed!